plugins/module_utils/network/junos/facts/acl_interfaces/acl_interfaces.py

```diff
@@ -93,13 +93,13 @@ def populate_facts(self, connection, ansible_facts, data=None):
 
         facts = {}
         if objs:
-            facts["junos_acl_interfaces"] = []
+            facts["acl_interfaces"] = []
             params = utils.validate_config(
                 self.argument_spec,
                 {"config": objs},
             )
             for cfg in params["config"]:
-                facts["junos_acl_interfaces"].append(utils.remove_empties(cfg))
```

Comment:
You should duplicate the setting instead of replacing it. So:

```
            facts["acl_interfaces"] = []
            # Included for compatibility, remove after 2025-07-01
            facts["junos_acl_interfaces"] = []
            params = utils.validate_config(
                self.argument_spec,
                {"config": objs},
            )
            for cfg in params["config"]:
                facts["acl_interfaces"].append(utils.remove_empties(cfg))
                # Included for compatibility, remove after 2025-07-01
                facts["junos_acl_interfaces"].append(utils.remove_empties(cfg))
```